JSDemos/Demos/Toolbar/Adaptability/Angular/app/app.component.html

```diff
@@ -69,6 +69,7 @@
           <dx-select-box
             placeholder="Font"
             displayExpr="text"
+            [inputAttr]="{ 'aria-label': 'Font' }"
```

Comment:
We use constant for inputAttr in the React demo: inputAttr={fontInputAttr}; but why you don't use the same object here? It's more consistent with another demos and data will be stored separately from templates.

Reply:
"why we don't use the same object here" - this object is within the file in react folder

Reply:
particularly this const is declared inside react folder structure, but im working on demo accessibility now, mb it's worth it to use the same approach, i'll see into that in my next PR with demo accessability